README.md

```diff
@@ -351,7 +387,7 @@ To attempt an exchange to test compatibility, the following will be needed:
 * A small amount above dust value of the blockchain's token, Blocknet's token(BLOCK) for the exchange network fee, and token of a compatible blockchain to trade against, which can also be BLOCK. Depending on the token's unit value, $1-3 USD should work fine.
 * Run the full nodes for each of the blockchains being tested on a testnet Service Node with the composed configuration files. Contact a contributor for tBLOCK or see if the chains can be added to existing testnet Service Nodes.
 * Two instances of each blockchain used in testing must be run, with each having the respective configuration files. Exchanges cannot be taken by the client that created them.
```

Comment:
>  Two instances of each blockchain used in testing must be run, with each having the respective configuration files.

For me, this idea could be more clearly explained with a simple example, something like:
"To test COIN-A by trading it with COIN-B, XBridge clients must be running on *testnet* on two separate computers. Each of these 2 XBridge clients must be connected through proper `xbridge.conf` configurations to instances of both COIN-A and COIN-B. Also, COIN-A and COIN-B must be properly configured individually in their respective <wallet>.conf files."

Reply:
Reworked and improved. I've also added the parse-xbridge.py script to the repo.